modules/ebi-metagenomics/seqfu/check/meta.yml

```diff
@@ -0,0 +1,44 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "seqfu_check"
+description: Perform sanity checking of FASTQ files
```

Comment:
The description on the seqfu site is probably better?
"Evaluates the integrity of DNA FASTQ files."